boto3/s3/inject.py

```diff
@@ -159,8 +159,8 @@ def download_file(
     Usage::
 
         import boto3
-        s3 = boto3.resource('s3')
-        s3.meta.client.download_file('mybucket', 'hello.txt', '/tmp/hello.txt')
+        s3 = boto3.client('s3')
```

Comment:
While we're here can you fix these for `bucket_upload_file` and `bucket_download_file` too, Tim?

Reply:
Shouldn't the [bucket_upload_file](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/bucket/upload_file.html) and [bucket_download_file](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/bucket/download_file.html) examples stay the same since they're under the [Bucket](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/bucket/index.html) -> [Actions](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/bucket/index.html#actions) documentation for resources?

Reply:
Yep, you're right, that was an oversight. Should be good to go.